photon-core/src/main/java/org/photonvision/vision/aruco/ArucoDetectionResult.java

```diff
@@ -62,11 +50,11 @@ public int getId() {
     }
 
     public double getCenterX() {
```

Comment:
Can we document why the algorithm below is valid? Is the center of mass of a quadrilateral always at the mean coordinates of its corners?

Reply:
Maybe this isn't relevant, but which center of mass do we want? https://sites.math.washington.edu/~king/java/gsp/center-mass-quad.html

Reply:
Maybe we should change this (and maybe the umich code) to use the opencv image moments function like we do for blob targets?